src/components/tearComponent/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  ResponsiveImg,
+  Background,
+  StyledExternalWrapper,
+  StyledInternalWrapper,
+} from './index.styles'
+
+import { Tear } from './tears'
+
+const TornBackground = ({
+  backgroundImage,
+  topTear,
+  bottomTear,
+  children,
+  minHeight,
+}) => {
+  return (
+    <StyledExternalWrapper>
```

Comment:
Might be an idea to wrap this in `section` tags. Then we could update the name of the component to `TornSection` which would make more sense semantically.